sys/newlib_syscalls_default/syscalls.c

```diff
@@ -662,3 +647,20 @@ clock_t _times_r(struct _reent *ptr, struct tms *ptms)
 
     return (-1);
 }
+
+/**
+ * @brief Wrapper for newlib's assert implementation
+ */
+NORETURN
+void __wrap___assert_func (const char *file, int line, const char *func, const char *expr)
+{
+    (void)file;
+    (void)line;
+    (void)func;
+    (void)expr;
+#if define(DEBUG_ASSERT_VERBOSE) && !defined(NDEBUG)
```

Comment:
```suggestion
#if defined(DEBUG_ASSERT_VERBOSE) && !defined(NDEBUG)
```

Typo from @benpicco :D